docs/whats_new.md

```diff
@@ -235,6 +235,28 @@ go app.Listen(":3000")
 
 This change simplifies the shutdown handling by consolidating the shutdown callbacks into a single hook that receives the error status.
 
+- Added Unix socket support
```

Comment:
Replace this with:

-  Added Unix socket support
+ - Added support for Unix domain sockets via `ListenerNetwork` and `UnixSocketFileMode`